test/vagrant-service-manager/binary_handlers/adb_docker_binary_handler_test.rb

```diff
@@ -54,6 +54,11 @@ module ServiceManager
       end
 
       it 'should validate download url' do
+        @handler.build_download_url
+        assert_silent { @handler.validate_url }
```

Comment:
> What will be method signature here ? def validate_url(url) ? This doesn't seems to be object oriented.

It is object oriented and increased test-ability atm. 

> Thought of that but then we are capturing the return value in install method. Whats its purpose of returning?

A more useful method and test-ability

> is fine enough and should not throw any kind of exception against what it is expected to have some exception on invalid URL which is covered by the test case just below this test case.

Sure, but what you are asserting is that the command is silent. That's about it. I guess what you are after is _assert_nothing_raised_. And if you use this assertion, you really want to assert the exceptional case as well. 

That said, why don't you add a Cucumber test for the oc cli install? This will once and for all show whether this works end to end.